pkgs/tools/networking/networkmanager/openconnect/default.nix

```diff
@@ -42,8 +42,9 @@ stdenv.mkDerivation rec {
     libxml2
     openconnect
     networkmanager
-  ] ++ lib.optionals withGnome [
     gtk3
+    webkitgtk # non-optional, requires gtk3
```

Comment:
Hmm, looks like it is needed for SSO: https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/-/commit/1e86b388f190e556aa9b934517b3a26cc64959d3

Also `webkitgtk_4_1` should probably be used.

Reply:
Thanks for bringing that up, I didn't realise that 4.1 was also already packaged. Should've taken better care there, sorry.